nil-docs/zkllvm/getting-started/writing-a-simple-circuit.mdx

```diff
@@ -0,0 +1,86 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Writing a simple circuit
+
+This tutorial describes how to create a simple circuit containing one multiplicaton operation.
+
+:::tip
+
+The C++ examples in this tutorial and across the rest of the documentation use types and algorithms from [**the `crypto3` library**](https://github.com/NilFoundation/crypto3).
+
+The Rust examples use types from [**the `arkworks-rs` project**](https://github.com/arkworks-rs).
+
+- [**`arkworks-algebra`**](https://github.com/NilFoundation/arkworks-algebra.git)
+- [**`arkworks-curves`**](https://github.com/NilFoundation/arkworks-curves.git)
+
+:::
+
+## Multiplication example
+
+Any circuit is a series of polynomials which, in the case of high-level programming languages, can be expressed as a function.
+
+For zkLLVM to correctly recognise a function as a circuit, it needs to be preceeded by the following directive.
+
+<Tabs groupId='language'>
+  <TabItem value='cpp' label='C++'>
+
+    ```cpp
+    [[circuit]]
+    ```
+
+  </TabItem>
+
+  <TabItem value='rust' label='Rust'>
+
+    ```rust
+    #[circuit]
+    ```
+
+  </TabItem>
+</Tabs>
+
+A simple example of a circuit with one multiplication operator would look as follows.
+
+<Tabs groupId='language'>
+  <TabItem value='cpp' label='C++'>
+
+    ```cpp
+
+    [[circuit]] int multiplication_example(
+    int a,
+    int b) {
+      auto c = a*b;
+      return c;
+    }
+
+    ```
+
+  </TabItem>
+
+  <TabItem value='rust' label='Rust'>
+
+    ```rust
+    #[circuit]
+    pub fn multiplication_example(a: i32, b: i32) -> i32 {
+      let c = a * b;
+      c
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+:::info[Using standard types in C++]
+
+Note that `clang` presently does not support `std::string` and several other types from `std`.
+
+:::
+
+The function takes two variables, multiplies them, and returns the result. With no additional code needed, it is a fully-fledged circle.
```

Comment:
"circle" is a typo?

Reply:
not sure what you mean, it is definitely needed here

Reply:
I thought you mean "fully-fledged circuit" here

Reply:
fair, changed the wording